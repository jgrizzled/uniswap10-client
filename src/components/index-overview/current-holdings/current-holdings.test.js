import CurrentHoldings from '.';

const holdings = mockData.holdingsByDate[0];
const tokens = mockData.tokens;
const component = <CurrentHoldings holdings={holdings} tokens={tokens} />;

// smoke test
it('renders without crashing', async () => {
  const div = document.createElement('div');
  ReactDOM.render(<Providers>component</Providers>, div);
  ReactDOM.unmountComponentAtNode(div);
});

// shallow snapshot test
it('renders as expected', async () => {
  const wrapper = Enzyme.shallow(component, { wrappingComponent: Providers });
  expect(wrapper).toMatchSnapshot();
});
