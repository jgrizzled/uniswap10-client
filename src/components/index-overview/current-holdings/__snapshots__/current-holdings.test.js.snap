// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<styled.div>
  <styled.div>
    <styled.h2>
      Current Holdings
    </styled.h2>
    <styled.div>
      <styled.ol>
        <li
          key="0"
        >
          <img
            alt="MKR"
            height="20px"
            src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2/logo.png"
          />
           
          <span>
            <a
              href="https://uniswap.info/token/0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2"
              target="blank"
            >
              MKR
            </a>
            : 
            50
            %
          </span>
        </li>
        <li
          key="1"
        >
          <img
            alt="ZRX"
            height="20px"
            src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xE41d2489571d322189246DaFA5ebDe1F4699F498/logo.png"
          />
           
          <span>
            <a
              href="https://uniswap.info/token/0xe41d2489571d322189246dafa5ebde1f4699f498"
              target="blank"
            >
              ZRX
            </a>
            : 
            50
            %
          </span>
        </li>
      </styled.ol>
    </styled.div>
  </styled.div>
</styled.div>
`;
